docs/integrations/app-integrations/gong.mdx

```diff
@@ -0,0 +1,272 @@
+    PARAMETERS = {
+        "access_key": "your_access_key",
+        "secret_key": "your_secret_key"
+    };
+```
+
+Required connection parameters include the following:
+
+**Authentication (choose one method):**
+
+* `api_key`: Bearer token for authentication (recommended)
+* `access_key` + `secret_key`: Basic authentication credentials (alternative method)
+
+Optional connection parameters include the following:
+
+* `base_url`: Gong API base URL. This parameter defaults to `https://api.gong.io`.
+* `timeout`: Request timeout in seconds. This parameter defaults to `30`.
+
+<Note>
+If both authentication methods are provided, basic auth (`access_key` + `secret_key`) takes precedence.
+</Note>
+
+## Usage
+
+The following usage examples utilize `gong_datasource` as the datasource name, which is defined in the `CREATE DATABASE` command.
+
+### Available Tables
+
+The Gong handler provides access to the following tables:
+
+* `calls` - Access call recordings and metadata
+* `users` - Get user information and permissions
+* `analytics` - Access AI-generated conversation insights
+* `transcripts` - Get full conversation transcripts
+
+### Basic Queries
+
+Retrieve recent calls with date filters (recommended for best performance):
+
+```sql
+SELECT * 
+FROM gong_datasource.calls 
+WHERE date >= '2024-01-01' AND date < '2024-02-01'
+ORDER BY date DESC
+LIMIT 20;
+```
+
+Get all users in your organization:
+
+```sql
+SELECT user_id, name, email, role, status 
+FROM gong_datasource.users 
+LIMIT 100;
+```
+
+Get analytics for calls with high sentiment scores:
+
+```sql
+SELECT call_id, sentiment_score, key_phrases, topics 
+FROM gong_datasource.analytics 
+WHERE sentiment_score > 0.7 
+  AND date >= '2024-01-01'
+LIMIT 50;
+```
+
+Get transcripts for a specific call:
+
+```sql
+SELECT speaker, timestamp, text 
+FROM gong_datasource.transcripts 
+WHERE call_id = '12345'
+ORDER BY timestamp;
+```
+
+### Advanced Queries with JOINs
+
+Get calls with their sentiment analysis:
+
+```sql
+SELECT 
+    c.title,
+    c.date,
+    c.duration,
+    a.sentiment_score,
+    a.key_phrases
+FROM gong_datasource.calls c
+JOIN gong_datasource.analytics a ON c.call_id = a.call_id
+WHERE c.date >= '2024-01-01' AND c.date < '2024-02-01'
+ORDER BY a.sentiment_score DESC
+LIMIT 25;
+```
+
+Find calls where specific keywords were mentioned:
+
+```sql
+SELECT 
+    c.title,
+    c.date,
+    t.speaker,
+    t.text
+FROM gong_datasource.calls c
+JOIN gong_datasource.transcripts t ON c.call_id = t.call_id
+WHERE c.date >= '2024-01-01'
+  AND t.text LIKE '%pricing%'
+LIMIT 50;
+```
+
+Get user performance with call sentiment:
+
+```sql
+SELECT 
+    u.name,
+    u.email,
+    c.call_id,
+    c.title,
+    a.sentiment_score
+FROM gong_datasource.users u
+JOIN gong_datasource.calls c ON u.user_id = c.user_id
+JOIN gong_datasource.analytics a ON c.call_id = a.call_id
+WHERE c.date >= '2024-01-01'
+  AND a.sentiment_score > 0.8
+LIMIT 100;
+```
+
+### Querying the Data Catalog
```

Comment:
These should be 'META_' tables, right? Like `META_TABLES`, `META_COLUMNS` and so forth. Is there a reason to even include these, @ZoranPandovski? Since they are mostly for internal use?